examples/complex_fluids/ex2/InterpolationUtilities.h

```diff
@@ -0,0 +1,80 @@
+#ifndef included_InterpolationUtilities
```

Comment:
Could this go into IBTK?

It would be good to have something that lets us easily interpolate fields at arbitrary locations in the main library.

Reply:
It could. It would need to be cleaned up a bit. I can create another PR for that. It also didn't pan out well for flow past a confined cylinder, although I think that's just the nature of that specific problem.

Reply:
@abarret see #776.